src/Aspire.Hosting.Sdk/Aspire.RuntimeIdentifier.Tool/Program.cs

```diff
@@ -0,0 +1,88 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System.CommandLine;
+using System.CommandLine.Parsing;
+using System.Diagnostics;
+using System.Reflection;
+using Aspire.Hosting.Sdk;
+using NuGet.RuntimeModel;
+
+namespace Aspire.RuntimeIdentifier.Tool;
+
+sealed class Program
+{
+    static int Main(string[] args)
+    {
+        CliRootCommand rootCommand = new("Aspire.RuntimeIdentifier.Tool v" + FileVersionInfo.GetVersionInfo(Assembly.GetExecutingAssembly().Location).ProductVersion)
+        {
+            TreatUnmatchedTokensAsErrors = true
+        };
+
+        CliOption<string?> runtimeGraphPathOption = new("--runtimeGraphPath", "-rgp")
+        {
+            Description = "Path to runtime graph path to use for RID mapping.",
+            Required = true
+        };
+
+        CliOption<string?> netcoreSdkRuntimeIdentifierOption = new("--netcoreSdkRuntimeIdentifier", "-r")
+        {
+            Description = "RID to use for finding the best applicable RID from mapping.",
+            Required = true
+        };
+
+        CliOption<string[]> supportedRidsOption = new("--supportedRids", "-sr")
+        {
+            Description = "List of RIDs that are supported. Comma-separated.",
+            Required = true,
+            AllowMultipleArgumentsPerToken = true,
```

Comment:
I don't think we need this. From the docs:

```
        /// <example>
        /// If set to <see langword="true"/>, the following command line is valid for passing multiple arguments:
        /// <code>
        /// > --opt 1 2 3
        /// </code>
        /// The following is equivalent and is always valid:
        /// <code>
        /// > --opt 1 --opt 2 --opt 3
        /// </code>
        /// </example>
```

Reply:
I remember trying without this and things didn't work, so not sure what was wrong in that case.